docs/Using-Fleet/REST-API.md

```diff
@@ -5706,39 +5714,45 @@ _Available in Fleet Premium_
 
 ```json
 {
-  "teams": [
-    {
-      "name": "workstations",
-      "id": 1,
-      "user_ids": [],
-      "host_ids": [],
-      "user_count": 0,
-      "host_count": 0,
-      "agent_options": {
-        "spec": {
-          "config": {
-            "options": {
-              "logger_plugin": "tls",
-              "pack_delimiter": "/",
-              "logger_tls_period": 10,
-              "distributed_plugin": "tls",
-              "disable_distributed": false,
-              "logger_tls_endpoint": "/api/v1/osquery/log",
-              "distributed_interval": 10,
-              "distributed_tls_max_attempts": 3
-            },
-            "decorators": {
-              "load": [
-                "SELECT uuid AS host_uuid FROM system_info;",
-                "SELECT hostname AS hostname FROM system_info;"
-              ]
-            }
+  "team": {
```

Comment:
I'm unclear on why this changed. I thought this might be causing the E2E failures, but when I hit this endpoint it's still returning the the `teams` array, whereas this doc change indicates that single `team` object is returned, which doesn't appear to be happening. 

It's okay for us to add the `webhook_settings` object here, but changing existing properties will create a breaking change that we're not prepared for yet.

Reply:
Good catch, I missed this.

Reply:
This was a copy paste issue in the docs. Thanks for catching this.